src/ActionModal/components/SendModal.vue

```diff
@@ -271,20 +275,6 @@ export default {
         )
       }
     },
-    setTokenAndBalance() {
```

Comment:
isn't this needed for multi denom support?

Reply:
this is a computed now